### ISSUE TITLE
The third terminal onwards won't accept input without being manually focused

### ISSUE THREAD

**Issue by LSLeary** [cdepillabout/termonad#45]
This looks like it might be related to #7, but it's not just cosmetic.

Observed behaviour:
 1. Open termonad. Terminal has keyboard focus and shows it. Optionally enter text.
 2. C-S-t open another terminal in a new tab. It has keyboard focus but doesn't show it. Optionally: enter text, gain the appearance of focus by refocusing the application window, or C-S-w close the terminal and C-S-t reopen it.
 3. C-S-t open another new terminal. It doesn't have keyboard focus. Refocusing the application window has no effect, nor does clicking on the tab (even if you click on another tab first), but clicking on the terminal text-box does.
 4. C-S-t open another new terminal. It doesn't have keyboard focus either. C-S-w close the terminal, hence arriving on the previous which now does have focus.

I haven't tested this with older termonad versions; it could be a recent regression.

**Comment by Grendel-Grendel-Grendel** [comment]
On my machine, this bug doesn't happen, and neither does #7
